src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayController.cs

```diff
@@ -172,11 +174,34 @@ public async Task<IActionResult> EvaluateLegalNamePage(LegalNamePageViewModel vi
             return RedirectToAction("ViewApplication", new { viewModel.ApplicationId });
         }
 
-        [HttpGet("/Roatp/Gateway/{applicationId}/Page/1-10")]
-        public async Task<IActionResult> GetGatewayLegalNamePage(Guid applicationId, string pageId)
+        [HttpGet("/Roatp/Gateway/{applicationId}/Page/1-20")]
+        public async Task<IActionResult> GetGatewayTradingNamePage(Guid applicationId, string pageId)
```

Comment:
Where's pageId coming from?  It needs to be decorated with [FromBody] if not coming from the URL Route